### PR TITLE
Update pom.xml to fix build errors

### PR DESCRIPTION
**Pull request by averikitsch**

### CODE REVIEW

File: appengine-java11/quarkus-helloworld/pom.xml [hunk 1]
```diff
@@ -29,8 +29,10 @@
   </parent>
 
   <properties>
+    <surefire-plugin.version>2.22.0</surefire-plugin.version>
     <maven.compiler.target>11</maven.compiler.target>
     <maven.compiler.source>11</maven.compiler.source>
+    <quarkus.version>0.13.1</quarkus.version>
```
Comment (ludoch) [inline appengine-java11/quarkus-helloworld/pom.xml]: Why not https://github.com/quarkusio/quarkus/releases/tag/1.2.0.Final ?

Comment (averikitsch) [reply appengine-java11/quarkus-helloworld/pom.xml]: Thanks for catching this!

Comment (ludoch) [reply appengine-java11/quarkus-helloworld/pom.xml]: Thanks... Did you deploy/access it for sanity check?

Comment (averikitsch) [reply appengine-java11/quarkus-helloworld/pom.xml]: Yes, good to go

**Comment by ludoch** [comment]
LGTM

**Pull request merged**